addons/compat_rhs_afrf3/compat_rhs_afrf3_csw/config.cpp

```diff
@@ -0,0 +1,45 @@
+#include "script_component.hpp"
+
+#if __has_include("\rhsafrf\addons\rhs_main\loadorder\config.bin")
+#else
+    #define PATCH_SKIP "RHS AFRF"
+#endif
+
+#if __has_include("\z\ace\addons\csw\script_component.hpp")
+#else
+    #ifdef PATCH_SKIP
+    #else
+        #define PATCH_SKIP "ACE Crew Served Weapons"
+    #endif
+#endif
+
+#ifdef PATCH_SKIP
+ACE_PATCH_NOT_LOADED(ADDON,PATCH_SKIP)
+#else
+
+#include "\z\ace\addons\csw\script_config_macros_csw.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = QUOTE(COMPONENT);
+        units[] = {};
+        weapons[] = {QGVAR(2b14_carry), QGVAR(nsv_carry), QGVAR(kord_carry), QGVAR(ags30_carry), QGVAR(spg9_carry), QGVAR(spg9m_carry), QGVAR(metis_carry), QGVAR(kornet_carry)};
```

Comment:
missions will be expecting old classnames like 
`ace_compat_rhs_afrf3_kornet_carry`
this changes classnames to
`ace_compat_rhs_afrf3_csw_kornet_carry`

changing to `EGVAR(compat_rhs,kornet_carry)` should keep old behaviour

Reply:
Redefined GVAR macro in root so GVAR is always `rhs_compat_afrf3`.